YogaKit/Source/YGLayout.h

```diff
@@ -149,3 +165,7 @@ typedef NS_OPTIONS(NSInteger, YGDimensionFlexibility) {
 - (void)markDirty;
 
 @end
+
+@interface YGLayoutContainer : YGLayout <YGLayoutEntity>
```

Comment:
The more I think about it, The more it makes sense to me that the `YGLayoutContainer` should be it's own NSObject.